src/main/java/org/apache/sling/feature/cpconverter/handlers/XmlConfigurationEntryHandler.java

```diff
@@ -65,13 +74,77 @@ protected void onJcrRootElement(String uri, String localName, String qName, Attr
 
                     if (attributeValue != null && !attributeValue.isEmpty()) {
                         DocViewProperty property = DocViewProperty.parse(attributeQName, attributeValue);
-
+                        Object value = property.values;
+                        List<String> strValues = Arrays.asList(property.values);
+                        switch (property.type) {
+                            case PropertyType.DATE:
+                                // Date was never properly supported as osgi configs don't support dates so converting to millis 
+                                // Scenario should just be theoretical
+                                attributeQName += ":Long";
+                                value = Lists.transform(strValues, new Function<String, Long>() {
+                                   public Long apply(String s) {
+                                      Long res = null;
+                                      if (s != null) {
+                                           Calendar cal = ISO8601.parse(s);
+                                           if (cal != null) {
+                                               res = cal.getTimeInMillis();
+                                           }
+                                      } 
+                                      return res;
+                                   }
+                                }).toArray();
+                                break;
+                            case PropertyType.DOUBLE:
+                                attributeQName += ":Double";
+                                value = Lists.transform(strValues, new Function<String, Double>() {
+                                   public Double apply(String s) {
+                                      Double res = null;
+                                      try {
+                                          res = Double.parseDouble(s);
+                                      } catch ( NumberFormatException ex ){
+                                          // ignore as nonexisting
```

Comment:
Are you sure that we should just skip invalid values for doubles and longs and not fail the conversion process?

Reply:
@rombert  - as those are typed representations (it is a string representation of a  typed property in the end) the only "valid" case of not being a number is an unset value. Therefore I saw no scenario where an explicit handling would do anything here.

Reply:
Wouldn't the below OSGi config snippet be silently ignored with this change?

```xml
  alertThreshold="{Double}high"
```

?

I think such invalid inputs should halt the conversion process.

Reply:
@rombert ok - in theory they shouldn't exist but yes you are right if somebody crafted  a broken vault package that may happen - I can make this throw an exception.

Reply:
Great, thanks!